src/integration-test/java/com/commercetools/sync/integration/externalsource/customobjects/CustomObjectSyncIT.java

```diff
@@ -115,133 +108,6 @@ void sync_withExistingCustomObjectThatHasSameValue_shouldNotUpdateCustomObject()
-
-        assertThat(customObjectSyncStatistics).hasValues(1, 0, 0, 1);
-        Assertions.assertThat(errorCallBackMessages).hasSize(1);
-        Assertions.assertThat(errorCallBackExceptions).hasSize(1);
-        Assertions.assertThat(errorCallBackMessages.get(0)).contains(
-            format("Failed to update custom object with key: '%s'. Reason: Failed to fetch from CTP while retrying "
-                + "after concurrency modification.", CustomObjectCompositeIdentifier.of(customObjectDraft)));
-    }
-
-    @Test
-    void sync_withConcurrentModificationExceptionAndUnexpectedDelete_shouldFailToReFetchAndUpdate() {
-
-        final SphereClient spyClient = spy(CTP_TARGET_CLIENT);
-
-        final CustomObjectUpsertCommand customObjectUpsertCommand = any(CustomObjectUpsertCommand.class);
-        when(spyClient.execute(customObjectUpsertCommand))
-                .thenReturn(CompletableFutureUtils.exceptionallyCompletedFuture(new ConcurrentModificationException()))
-                .thenCallRealMethod();
-
-        final CustomObjectQuery customObjectQuery = any(CustomObjectQuery.class);
-
-        when(spyClient.execute(customObjectQuery))
-                .thenCallRealMethod()
-                .thenReturn(CompletableFuture.completedFuture(PagedQueryResult.empty()));
-
-        final ObjectNode newCustomObjectValue = JsonNodeFactory.instance.objectNode().put("name", "value2");
-        List<String> errorCallBackMessages = new ArrayList<>();
-        List<Throwable> errorCallBackExceptions = new ArrayList<>();
-
-        final CustomObjectSyncOptions spyOptions = CustomObjectSyncOptionsBuilder
-            .of(spyClient)
-            .errorCallback((exception, oldResource, newResource, updateActions) -> {
-                errorCallBackMessages.add(exception.getMessage());
-                errorCallBackExceptions.add(exception.getCause());
-            })
-            .build();
-
-        final CustomObjectSync customObjectSync = new CustomObjectSync(spyOptions);
-
-        final CustomObjectDraft<JsonNode> customObjectDraft = CustomObjectDraft.ofUnversionedUpsert("container1",
-            "key1", newCustomObjectValue);
-
-        final CustomObjectSyncStatistics customObjectSyncStatistics = customObjectSync
-            .sync(Collections.singletonList(customObjectDraft))
-            .toCompletableFuture().join();
-
-        assertThat(customObjectSyncStatistics).hasValues(1, 0, 0, 1);
-        Assertions.assertThat(errorCallBackMessages).hasSize(1);
-        Assertions.assertThat(errorCallBackExceptions).hasSize(1);
-
-        Assertions.assertThat(errorCallBackMessages.get(0)).contains(
-            format("Failed to update custom object with key: '%s'. Reason: Not found when attempting to fetch while"
-                + " retrying after concurrency modification.", CustomObjectCompositeIdentifier.of(customObjectDraft)));
-    }
-
     @Test
     void sync_withNewCustomObjectAndBadRequest_shouldNotCreateButHandleError() {
```

Comment:
Do you think that we need to move this into unit tests? 🤔

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/636/commits/c1e492b1dfa42b4f1ed55e9adb5c5bb88db4e9d7